plugins/ccgx/ccgx-dock.quirk

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (C) 2020 Cypress Semiconductor Corporation.
+#
+# SPDX-License-Identifier: LGPL-2.1+
+#
+
+#######################################################
+## Gen2 Dock
+#######################################################
+[DeviceInstanceId=USB\VID_17EF&PID_A38F]
+Name = Lenovo Gen2 Dock Billboard
+Plugin = ccgx
+GType = FuCcgxDockBb
+ParentGuid = USB\VID_17EF&PID_A392
+
+[DeviceInstanceId=USB\VID_04B4&PID_521A]
+Name = Lenovo Gen2 Dock USB Serial Bridge
+Plugin = ccgx
+GType = FuCcgxDockBb
+ParentGuid = USB\VID_17EF&PID_A392
```

Comment:
shouldn't this be `cy-device-external-bb` too?

Reply:
Only Billboard Device (VID 17EF, PID A392) have cy-device-external-bb flag.
but USB Serial Bridge is not Billboard device.